lib/generators/rodauth/install_generator.rb

```diff
@@ -70,6 +70,10 @@ def create_mailer
           end
         end
 
+        def create_fixtures
+          template "app/test/fixtures/accounts.yml"                
+        end
```

Comment:
Would it be possible to detect when fixture generation has been disabled at the Rails app level ([StackOverflow answer](https://stackoverflow.com/a/8497702/988414)), and skip generating the fixture?

Reply:
I"ll have to look more deeply into this one. I couldn't find a quick fix answer. please bear with me.

Reply:
FWIW, there is already code in the install generator that reads something from `generators` Rails configuration. Cannot link to the line since I'm on mobile.

Reply:
If it's there then i'll find it. no need to search the link.